stories/required_with_context/ComponentWithRef.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react'
```

Comment:
Does this project use React v15.5? If so it should bring in `prop-types` separately [and if not ignore this]